gum/arch-mips/gummipsreader.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2015 Ole André Vadla Ravnås <oleavr@nowsecure.com>
+ *
+ * Licence: wxWindows Library Licence, Version 3.1
+ */
+
+#include "gummipsreader.h"
+
+#include <capstone.h>
+
+static cs_insn * disassemble_instruction_at (gconstpointer address);
+
+gpointer
+gum_arm64_reader_try_get_relative_jump_target (gconstpointer address)
```

Comment:
arm64? I suppose we should remove the implementations here and just `g_assert_not_reached ()` until it's implemented.